### PR TITLE
Rh starting on UI

### PR DESCRIPTION
**Pull request by rezahusain**

### CODE REVIEW

**Review by jb520 — APPROVED** [review]
Everything looks good to me!

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class ApplicationController < ActionController::Base
   before_action :configure_permitted_parameters, if: :devise_controller?
+  before_action :authenticate_user!
```
Comment (JoseMDO) [inline app/controllers/application_controller.rb:3]: Nice!

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -3,16 +3,30 @@
   <head>
     <title>Photogram Industrial</title>
     <meta name="viewport" content="width=device-width,initial-scale=1">
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
+    <%= render partial: "shared/cdn_assets" %>
```
Comment (JoseMDO) [inline app/views/layouts/application.html.erb:9]: Great job adding your CDN assets to a partial! This helps keep your layout file clean :)

File: app/views/shared/_navbar.html.erb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    </button>
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav mr-auto">
+        <li class="nav-item dropdown">
+          <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+            Resources
+          </a>
+          <div class="dropdown-menu">
+            <a class="dropdown-item" href="/photos">Photos</a>
+            <a class="dropdown-item" href="/comments">Comments</a>
```
Comment (JoseMDO) [inline app/views/shared/_navbar.html.erb:16]: remember you can start using link_to tags now :)

File: lib/tasks/dev.rake [hunk 4]
```diff
@@ -8,6 +8,20 @@ task sample_data: :environment do
   Photo.delete_all
   User.delete_all
 
+  usernames = Array.new { Faker::Name.first_name }
+
+  usernames << "alice"
+  usernames << "bob"
+
+  usernames.each do |username|
```
Comment (JoseMDO) [inline lib/tasks/dev.rake:16]: nice job on your sample_data!

**Review by JoseMDO — APPROVED** [review]
Great job Reza!

**Pull request merged**